lib/core-server/src/dev-server.ts

```diff
@@ -1,8 +1,14 @@
 import express, { Router } from 'express';
 import compression from 'compression';
 
-import type { CoreConfig, Options, StorybookConfig } from '@storybook/core-common';
-import { normalizeStories, logConfig } from '@storybook/core-common';
+import {
```

Comment:
This goes against the coding style established by @ndelangen where the types & concrete imports are always separate. We should update it OR update the project style guidelines